walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -311,4 +313,167 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalIsBusy() {
```

Comment:
nit: maybe name this optionalAttendeeIsBusy()

Reply:
good idea, fixed this